app/components/component-docs/ComponentDocs.js

```diff
@@ -0,0 +1,37 @@
+// eslint-disable-next-line no-unused-vars, unused-imports/no-unused-imports
+import React from "react";
+
+import { CoreClasses, CoreH4, CoreTypographyBody1 } from "@wrappid/core";
+
+import CodeImport from "../CodeImport";
+import ComponentProps from "../ComponentProps";
+
+function ComponentDocs(props) {
+  const { component = null, description = null, samples = [] } = props;
+
+  return (
+    <>
+      <CoreH4 styleClasses={[CoreClasses.MARGIN.MY2, CoreClasses.COLOR.TEXT_PRIMARY]}>
+        {component?.name || "Name not provided in the component."}
+      </CoreH4>
+
+      <CoreTypographyBody1>{description || "Description not provided."}</CoreTypographyBody1>
+
+      <CodeImport name={component?.name} />
```

Comment:
change name to displayName

Reply:
no need to do that as of now

Reply:
Why?
ComponentDocs should show component.displayName instead of component.name.

This will also verify if all CoreComponents have the displayName.